components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/FirefoxAccount.kt

```diff
@@ -264,6 +264,21 @@ class FirefoxAccount(handle: FxaHandle, persistCallback: PersistCallback?) : Aut
         }.getAndConsumeRustString()
     }
 
+    /**
+     * Returns an OAuth code for a given client id and scopes. Requires a session token
+     * to be set on the account.
+     *
+     * @param clientId OAuth client id
+     * @param scopes Array of OAuth scopes
+     * This performs network requests, and should not be used on the main thread.
+     */
+    fun authorizedAuthCode(clientId: String, scopes: Array<String>): String {
```

Comment:
My bad for not proof-reading my own earlier comment about naming, but this should be present-tense "authorize" rather than past-tense "authorized". Sorry!